packaging/fedora/charliecloud.spec

```diff
@@ -46,53 +29,120 @@ a standard Linux filesystem tree.
 
 For more information: https://hpc.github.io/charliecloud/
 
+%package        doc
+Summary:        Charliecloud html documentation
+License:        BSD and MIT and ASL 2.0
+BuildArch:      noarch
+Obsoletes:      %{name}-doc < %{version}-%{release}
+%if 0%{?el7}
+BuildRequires:  python36-sphinx
+BuildRequires:  python36-sphinx_rtd_theme
+%else
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_rtd_theme
+%endif
+
+%description doc
+Html and man page documentation for %{name}.
+
+%package        test
+Summary:        Charliecloud test suite
+License:        ASL 2.0
+Obsoletes:      %{name}-test < %{version}-%{release}
+Requires:       %{name} bash /usr/bin/bats /usr/bin/python3
+
 %description test
-Charliecloud test suite and examples. The test suite takes advantage of
-container image builders such as Docker, Skopeo, and Buildah.
+Test fixtures for %{name}.
 
 %prep
 %setup -q
 
-%{versionize_script python3 test/docs-sane}
-%{versionize_script python3 test/make-perms-test}
-
 %build
-%make_build CFLAGS="%build_cflags -std=c11 -pthread" LDFLAGS="%build_ldflags"
+
+%configure --prefix=%{_prefix} \
+           --libdir=%{_libdir} \
+           --with-python=/usr/bin/python3 \
+%if 0%{?el7}
+            --with-sphinx-build=%{_bindir}/sphinx-build-3.6
+%else
+            --with-sphinx-build=%{_bindir}/sphinx-build
+%endif
 
 %install
-%make_install PREFIX=%{_prefix}
+%make_install
 
 cat > README.EL7 <<EOF
 For RHEL7 you must increase the number of available user namespaces to a non-
 zero number (note the number below is taken from the default for RHEL8):
 
   echo user.max_user_namespaces=3171 >/etc/sysctl.d/51-userns.conf
-  systemctl -p
+  reboot
```

Comment:
I think this should read `sysctl -p` ([man page](http://man7.org/linux/man-pages/man8/sysctl.8.html)); a reboot is not required. `systemctl -p` was indeed incorrect and means something else.